kpops/components/streams_bootstrap/model.py

```diff
@@ -221,8 +222,8 @@ class StreamsBootstrapValues(HelmAppValues):
         description=describe_attr("affinity", __doc__),
     )
 
-    tolerations: list[Toleration] | None = Field(
-        default=None,
+    tolerations: list[Toleration] = Field(
```

Comment:
You need SerializeAsOptional here, right?

Reply:
yep, missed that one